superset-frontend/src/SqlLab/actions/sqlLab.test.js

```diff
@@ -415,10 +415,10 @@ describe('async actions', () => {
     fetchMock.delete(updateTableSchemaEndpoint, {});
     fetchMock.post(updateTableSchemaEndpoint, JSON.stringify({ id: 1 }));
 
-    const getTableMetadataEndpoint = 'glob:*/api/v1/database/*';
+    const getTableMetadataEndpoint = 'glob:**/api/v1/database/*/table/*/*/';
```

Comment:
isn't a single asterisk here sufficient?
```suggestion
    const getTableMetadataEndpoint = 'glob:*/api/v1/database/*/table/*/*/';
```